commercial/app/model/merchandise/Merchandise.scala

```diff
@@ -134,8 +120,29 @@ case class Job(id: Int,
   val mainIndustry: Option[String] = industries.headOption
 }
 
+sealed trait Gender {
+  val name: String
+}
+
+object Gender {
+
+  def fromName(name: String) = name match {
+    case Woman.name => Woman
+    case _ => Man
+  }
+}
+
+case object Woman extends Gender {
+  val name: String = "Woman"
+}
+
+case object Man extends Gender {
+  val name: String = "Man"
+}
+
 object Merchandise {
-  val writes: Writes[Merchandise] = new Writes[Merchandise] {
+
+  val merchandiseWrites: Writes[Merchandise] = new Writes[Merchandise] {
```

Comment:
This looks a bit odd.  I'd have thought books and jobs know how to represent themselves.